docs/root/version_history/current.rst

```diff
@@ -15,6 +15,7 @@ Minor Behavior Changes
 * access_log: log all header values in the grpc access log.
 * dynamic_forward_proxy: if a DNS resolution fails, failing immediately with a specific resolution error, rather than finishing up all local filters and failing to select an upstream host.
 * ext_authz: added requested server name in ext_authz network filter for auth review.
+* ext_authz: forward ``typed_filter_metadata`` selected by ``typed_metadata_context_namespaces`` to external auth service.
```

Comment:
please ref link to the fields in question, thank you

Reply:
Thanks, updated.